doc/userguide/rules/header-keywords.rst

```diff
@@ -495,43 +499,75 @@ The following lists the meaning of all ICMP types. When a code is not listed,
+|           +-----------+------------------------------------------------------------------------+
+|           | 7         |  Destination Host Unknown                                              |
+|           +-----------+------------------------------------------------------------------------+
+|           | 8         |  Source Host Isolated                                                  |
+|           +-----------+------------------------------------------------------------------------+
+|           | 9         |  Communication with Destination Network is Administratively Prohibited |
+|           +-----------+------------------------------------------------------------------------+
+|           | 10        |  Communication with Destination Host is Administratively Prohibited    |
+|           +-----------+------------------------------------------------------------------------+
+|           | 11        |  Destination Network Unreachable for Type of Service                   |
+|           +-----------+------------------------------------------------------------------------+
+|           | 12        |  Destination Host Unreachable for Type of Service                      |
+|           +-----------+------------------------------------------------------------------------+
+|           | 13        |  Communication Administratively Prohibited                             |
+|           +-----------+------------------------------------------------------------------------+
+|           | 14        |  Host Precedence Violation                                             |
+|           +-----------+------------------------------------------------------------------------+
+|           | 15        |  Precedence cutoff in effect                                           |
++-----------+-----------+------------------------------------------------------------------------+
+| 5         | 0         |  Redirect Datagram for the Network (or subnet)                         |
+|           +-----------+------------------------------------------------------------------------+
+|           | 1         |  Redirect Datagram for the Host                                        |
+|           +-----------+------------------------------------------------------------------------+
+|           | 2         |  Redirect Datagram for the Type of Service and Network                 |
+|           +-----------+------------------------------------------------------------------------+
+|           | 3         |  Redirect Datagram for the Type of Service and Host                    |
+|           +-----------+------------------------------------------------------------------------+
+|           | 0         |  Normal router advertisement                                           |
+|           +-----------+------------------------------------------------------------------------+
+|           | 16        |  Doest not route common traffic                                        |
```

Comment:
I couldn't find `16` in the rfc ... `Doest` --> should be `Doesn't`?

Reply:
Good catch on 16, it was supposed to be code 9 !